critique.py

```diff
@@ -149,11 +181,12 @@ def get_default(group, subgroupclass):
         if self.openHDF5(fullname):
             entry = get_default(self.hdf5, 'NXentry')
             if entry:
-                data = get_default(entry, 'NXdata')
-                if data:
-                    if 'signal' in data.attrs:
-                        signalname = data.attrs['signal']
-                        compliance = signalname in data
+                compliance = True
+                # data = get_default(entry, 'NXdata')
```

Comment:
Remove commented out code and replace with comment. Also it seems that you return None